docs/schema.md

```diff
@@ -99,6 +104,30 @@ Representation of an AWS Account.
 	(AWSAccount)-[AWS_ROLE]->(AWSRole)
 	```
 
+## AWSCidrBlock
+### AWSIpv4CidrBlock
+### AWSIpv6CidrBlock
+Representation of an AWS CidrBlock used in VPC configuration. The `AWSCidrBlock` defines the base label
```

Comment:
Adding link: 

`Representation of an [AWS CidrBlock used in VPC configuration](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_VpcCidrBlockAssociation.html).`